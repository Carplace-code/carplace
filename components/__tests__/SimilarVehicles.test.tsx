// components/__tests__/SimilarVehicles.test.tsx
import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";

import SimilarVehicles from "../SimilarVehicles";

describe("SimilarVehicles", () => {
  it("renders correctly and matches snapshot", () => {
    const { container } = render(<SimilarVehicles />);
    expect(container).toMatchSnapshot();
  });
});
