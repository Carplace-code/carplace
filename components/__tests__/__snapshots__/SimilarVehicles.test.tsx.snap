// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SimilarVehicles > renders correctly and matches snapshot 1`] = `
<div>
  <div
    class="rounded-md border bg-white p-6"
  >
    <h2
      class="mb-4 text-lg font-semibold"
    >
      Similar Vehicles
    </h2>
    <ul
      class="space-y-3 text-sm"
    >
      <li
        class="flex items-start justify-between"
      >
        <div>
          <p
            class="font-medium"
          >
            Mercedes-Benz C200 2022
          </p>
          <p
            class="text-xs text-gray-500"
          >
            12,000 km
          </p>
        </div>
        <p
          class="font-bold text-blue-600"
        >
          $29,900,000
        </p>
      </li>
      <li
        class="flex items-start justify-between"
      >
        <div>
          <p
            class="font-medium"
          >
            Mercedes-Benz C200 2019
          </p>
          <p
            class="text-xs text-gray-500"
          >
            18,000 km
          </p>
        </div>
        <p
          class="font-bold text-blue-600"
        >
          $24,900,000
        </p>
      </li>
      <li
        class="flex items-start justify-between"
      >
        <div>
          <p
            class="font-medium"
          >
            Mercedes-Benz C200 2020
          </p>
          <p
            class="text-xs text-gray-500"
          >
            15,000 km
          </p>
        </div>
        <p
          class="font-bold text-blue-600"
        >
          $35,500,000
        </p>
      </li>
    </ul>
    <button
      class="mt-4 w-full rounded-md border py-1 text-sm hover:bg-gray-100"
      type="button"
    >
      Compare Selected
    </button>
  </div>
</div>
`;
